Add Pressable Component (#712)
reason-react-native/reason-react-native: src/components/Pressable.md

---
id: components/Pressable
title: Pressable

> wip: true

Reviewer: ```suggestion
officialDoc: https://reactnative.dev/docs/pressable
```

wip: is reserved to mostly empty markdown. Here you wrote enough to start :)

---

Pressable is a Core Component wrapper that can detect various stages of press interactions on any of it's defined children.

## How it works

> On an element wrapped by Pressable:

Reviewer: ```suggestion
On an element wrapped by `Pressable`:
```

> 1. onPressIn is called when a press is activated.
> 2. onPressOut is called when the press gesture is deactivated.

Reviewer: ```suggestion
1. `onPressIn` is called when a press is activated.
2. `onPressOut` is called when the press gesture is deactivated.
```

> You can read more about this [here](https://reactnative.dev/docs/pressable).

Reviewer: ```suggestion
You can read more about this [on the official Pressable documentation](https://reactnative.dev/docs/pressable).
```

title: Pressable
wip: true
---

Pressable is a Core Component wrapper that can detect various stages of press interactions on any of it's defined children.

## How it works

On an element wrapped by Pressable:

1. onPressIn is called when a press is activated.
2. onPressOut is called when the press gesture is deactivated.

> You can read more about this here.

Reviewer: Did you forget a link maybe?

Author: yep hahah. sorry about that.
